.env.example

```diff
@@ -14,3 +14,4 @@ REACT_APP_FORTMATIC_KEY=pk_test_886ADCAB855632AA
 REACT_APP_PORTIS_KEY=b2b7586f-2b1e-4c30-a7fb-c2d1533b153b
 
 REACT_APP_ZAPPER_API_KEY=96e0cc51-a62e-42ca-acee-910ea7d2a241
+REACT_APP_YEARN_SUBGRAPH_KEY=
```

Comment:
should we add a default one?

Reply:
Dont have one that I know of, but would be good to add later for devs.